README.md

```diff
@@ -63,11 +63,13 @@ Testing the Payroll Income or Document Income flows requires modifying the steps
 
 ### Transfer notes
 
-Using the Transfer endpoints requires you to be enrolled in the Transfer beta. Contact your Plaid Account manager or Plaid Sales to join the beta.
+To call Transfer endpoints, you will need either a payment profile token, or both an access token and an account id. 
 
-To call Transfer endpoints, you will need either a payment profile token, or both an access token and an account id. To obtain a payment profile token, call Transfer -> Create Payment Profile. To obtain an access token and account id, follow the [Making API calls in Sandbox](#making-api-calls-in-sandbox) steps above, then call Items -> Item Management -> Retrieve an Item's accounts. 
+The easiest method is to use an access token an account id. To obtain an access token and account id, follow the [Making API calls in Sandbox](#making-api-calls-in-sandbox) steps above, then call Items -> Item Management -> Retrieve an Item's accounts. 
 
-Once you have a payment profile token or an access token / account id pair, start by calling Transfer -> Authorize a transfer, then call Transfer -> Initiate a transfer. Note that you will need to update the request bodies for these endpoints to use only the account identifying mechanism you are choosing -- for example, if you are using Payment Profiles, delete the `account_id` and `access_token` fields before making the request.
+Once you have the access token / account id pair, start by calling Transfer -> Authorize a transfer, then call Transfer -> Initiate a transfer. Note that you will need to update the request bodies for these endpoints to use only the account identifying mechanism you are choosing -- for example, if you are using the account id and access token approach, delete the `payment_profile` field before making the request.
+
+If you want to use the Payment Profile approach instead of using an access token and account id, create the Payment Profile by calling Transfer -> Create Payment Profile.  If you are using this approach, you will not be able to complete the entire flow in Postman without using Link; after creating the payment profile token, you will need to activate the token by completing a Link session as described in [Making API calls with real data in Production or Development](#making-api-calls-with-real-data-in-production-or-development). You will only need to complete the process through step 12; it is not required to exchange the `public_token`. At step 6, where you call `/link/token/create` in Postman, you will need to add your payment profile token to the `/link/token/create` request body, by adding the following to the request body: `transfer: {payment_profile_token: your_payment_profile_token_goes_here}`. (Replace `your_payment_profile_token_goes_here` with the actual payment profile token returned by calling Create Payment Profile.) If you are not comfortable modifying request bodies in Postman, it is recommended to use the access token / account id approach instead.
```

Comment:
Will you also need to remove the account_id and access_token fields?

Reply:
Yeah, but I think we talk about that above -- basically this is the advanced approach, so I'm expecting folks who follow this step to be capable of following the reasoning about what they need to do next. The original organization had this paragraph before the instructions for calling Transfer -> Authorize a Transfer but I felt like it made more sense to optimize for making the easy path really easy. Also, the error messaging in the API if you send both fields is super clear and explicit about what the problem is, so I figure it won't be hard for people to self-correct if they get this wrong.